src/installer.rs

```diff
@@ -143,28 +153,26 @@ pub async fn install_client(client: Client, args: ClientInstallation) -> Result<
     // Create profile directory
     fs::create_dir_all(&profile_dir)?;
 
-    // Create an empty jar file to make the vanilla launcher happy
-    File::create(profile_dir.join(profile_name.clone() + ".jar"))?;
-
     // Create launch json
     let mut file = File::create(profile_dir.join(profile_name.clone() + ".json"))?;
 
     // Download launch json
-    let mut response = client.get(format!(
-        "https://meta.quiltmc.org/v3/versions/loader/{}/{}/profile/json",
-        &args.minecraft_version.version, &args.loader_version.version
-    ))
-    .send()
-    .await?
-    .text()
-    .await?;
+    let mut response = client
+        .get(format!(
+            "https://meta.quiltmc.org/v3/versions/loader/{}/{}/profile/json",
+            &args.minecraft_version.version, &args.loader_version.version
+        ))
+        .send()
+        .await?
+        .text()
+        .await?;
 
     // Hack-Fix:
     // Was fixed in versions above 0.17.7
     if args.loader_version.version < Version::new(0, 17, 7) {
-        // Quilt-meta specifies both hashed and intermediary, but providing both to quilt-loader causes it to silently fail remapping.
-        // This really shouldn't be fixed here in the installer, but we need a solution now.
-        let mut json: Value = serde_json::from_str(&response).unwrap();
+        // Quilt-meta specifies both hashed and intermediary,
+        // but providing both to quilt-loader causes it to silently fail remapping.
```

Comment:
maybe specify that this applies to old versions only.

Reply:
It is at [line 171](https://github.com/QuiltMC/quilt-native-installer/pull/20/files#diff-4904922dab3fd7bd66e213f49476ad3aa39a8a7c0cb24f68a68343909bbdaab4R171)